src/pages/index.js

```diff
@@ -0,0 +1,368 @@
+//   },
+
+//   {
+//     name: "Restaurant terrace",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//   },
+
+//   {
+//     name: "An outdoor cafe",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//   },
+
+//   {
+//     name: "A very long bridge, over the forest and through the trees",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//   },
+
+//   {
+//     name: "Tunnel with morning light",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//   },
+
+//   {
+//     name: "Mountain house",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+
+//   {
+//     name: "Golden Gate Bridge",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
+//   },
+// ];
+
+// Instantiating API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "506cab27-cb57-4614-a4cb-84697f37c55f",
+    "Content-Type": "application/json",
+  },
+});
+
+// Profile Elements
+const avatarModalButton = document.querySelector(".profile__avatar-button");
+const avatarImageElement = document.querySelector(".profile__avatar-image");
+const editProfileModalButton = document.querySelector(".profile__edit-button");
+const addCardModalButton = document.querySelector(".profile__add-button");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+
+// Modal Form Elements
+const modals = document.querySelectorAll(".modal");
+
+// Avatar Modal Form Elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarFormElement = avatarModal.querySelector(".modal__form");
+const avatarModalSubmitButton = avatarModal.querySelector(
+  ".modal__submit-button"
+);
+const avatarModalLinkInput = avatarModal.querySelector("#profile-avatar-input");
+
+// Edit Profile Modal Form Elements
+const editProfileModal = document.querySelector("#edit-modal");
+const editProfileFormElement = editProfileModal.querySelector(".modal__form");
+const editProfileModalNameInput = editProfileModal.querySelector(
+  "#profile-name-input"
+);
+const editProfileModalDescriptionInput = editProfileModal.querySelector(
+  "#profile-description-input"
+);
+
+// Add Card (aka New Post) Modal Form Elements
+const addCardModal = document.querySelector("#add-card-modal");
+const addCardFormElement = addCardModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms